joomla/joomla-cms
# [5.1] Article Schemaorg plugin

### Summary of Changes
Simple PR to add the most basic type of schema - Article



### Testing Instructions
Install prebuilt package or discover->install and publish plugin

Check that article is in the list of available schema in a content item
Add some values, save the article and then test the generated json-ld markup on the page at https://search.google.com/test/rich-results

### Link to documentations

Can't find any documentation for the schemaorg plugins. If you can point me at it then I can add this new schema 


Please select:
